plugins/calibre.koplugin/wireless.lua

```diff
@@ -499,7 +591,6 @@ function CalibreWireless:noop(arg)
     if arg.ejecting then
         self:sendJsonData('OK', {})
         self.disconnected_by_server = true
-        self:disconnect()
```

Comment:
is this no longer required? ie: when you desconnect from calibre side

Reply:
It's done in the connect coroutine.